irods_testing_environment/irods_setup.py

```diff
@@ -373,7 +405,8 @@ def setup_irods_catalog_consumer(ctx,
     )
 
     setup_input = (setup_input_builder()
-        .setup(catalog_service_role='consumer',
+        .setup(irods_version=irods_config.get_irods_version(csp_container),
```

Comment:
We will not be supporting heterogeneous zones, but for consistency, it might be worth moving the `catalog_consumer_container` instantiation up and then use that to get the version.

Reply:
Can you elaborate a little more?

Reply:
I'm saying we should get the version from the consumer since we are setting up the consumer in this method. That would require moving lines 417 through 422 up above this in order to have the consumer container.

However, since we are not going to support running various versions within the same zone - let alone various *major versions* - it doesn't really matter whether we get the version from the provider or the consumer. So this is fine as is, just thought it would be nice for consistency.

Reply:
That makes more sense to me and doesn't block the possibility of heterogeneous setups.

Now, I just need to figure out how to do that lol.

Reply:
Wait, I just noticed the lines you were referring to.

Reply:
I changed the name of the container variable to match the `csp_*` style naming.

Look good?

Reply:
Perfect